conseil-common/src/main/scala/tech/cryptonomic/conseil/common/bitcoin/BitcoinPersistence.scala

```diff
@@ -34,6 +35,20 @@ class BitcoinPersistence[F[_]: Concurrent] extends LazyLogging {
       Tables.Inputs ++= transactions.flatMap(t => t.vin.map(c => (t, c))).map(_.convertTo[Tables.InputsRow]),
       Tables.Outputs ++= transactions.flatMap(t => t.vout.map(c => (t, c))).map(_.convertTo[Tables.OutputsRow])
     )
+
+  /**
+    * Get sequence of existing blocks heights from the database.
+    *
+    * @param range Inclusive range of the block's height
+    */
+  def getIndexedBlockHeights(range: Range.Inclusive): FixedSqlStreamingAction[Seq[Int], Int, Effect.Read] =
```

Comment:
I think being so precise on the return value is not a good choice with a public API.

Would you agree on showing something generic as `DBIO[Seq[Int]]`?

Reply:
done